pkg/apis/core/v1alpha1/types.go

```diff
@@ -141,12 +143,25 @@ type DataVolumeList struct {
 	Items []DataVolume `json:"items"`
 }
 
+// DataVolumeCondition represents the state of a data volume condition.
+type DataVolumeCondition struct {
+	Type   DataVolumeConditionType `json:"type" description:"type of condition ie. Ready|Bound|Running."`
+	Status corev1.ConditionStatus  `json:"status" description:"status of the condition, one of True, False, Unknown"`
+	// +optional
+	Reason string `json:"reason,omitempty" description:"one-word CamelCase reason for the condition's last transition"`
```

Comment:
Why only one word?

Reply:
So I can easily copy to/from event reasons without having to modify anything.

Reply:
That sounds easy but maybe more detailed info would be helpful?

Reply:
Well the message has more detail.